incubator/kafka/Chart.yaml

```diff
@@ -3,7 +3,7 @@ description: Apache Kafka is publish-subscribe messaging rethought as a distribu
   commit log.
 name: kafka
 version: 0.20.7
```

Comment:
```suggestion
version: 0.20.8
```

Reply:
Can do. 
Unsure of your approach here; with a 3 version roll-forward of Kafka it could also be 21.0?

Reply:
Updated with 20.8